CHANGELOG.md

```diff
@@ -20,6 +20,9 @@ The project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
 ### Internal improvements
 
+- All Exonum crates have been updated to Rust 2018 edition. This means that
+  we require at least Rust 1.31 for compilation. (#1230)
```

Comment:
I have an information that even before this change build is impossible because some of our dependencies have already transfered to 2018 :wink:

Reply:
As additional note, I think we should not use pronouns in the changelog. So:
```suggestion
  it is required to use Rust 1.31 or newer for compilation. (#1230)
```